docs(project): update to reflect v10 end of support (#17615)
carbon-design-system/carbon: .github/CODE_OF_CONDUCT.md

    with the same questions can benefit.
4.  If you have a pressing bug fix, it is best to make a PR directly to get your
    issues addressed.

> 5.  `#carbon-react`, `#carbon-web-components`: This channel is for questions

Reviewer: ```suggestion
5.  `#carbon-react`, `#carbon-web-components`: These channels are for questions
```

Author: Great catch, thanks!

>     about Carbon Components only. It is not the place to ask general coding

Reviewer: ```suggestion
    about Carbon Components only. They are not the place to ask general coding
```
